src/trufflesom/interpreter/Invokable.java

```diff
@@ -20,22 +19,14 @@
   protected final Source source;
   protected final long   sourceCoord;
 
-  @Child protected ExpressionNode expressionOrSequence;
```

Comment:
What did that correspond to in the case of a Primitive, anyway? Was it used?

Reply:
This was the field where in a normal method you'd have the method body, and in a primitive you'd have a node that is annotated with the `@Primitive` annotation.

For example, for all the inner classes in https://github.com/SOM-st/TruffleSOM/blob/master/src/trufflesom/primitives/reflection/ClassPrims.java, we construct a `Primitive` object, which is plugged into the `Class` class for the methods identified by the annotation.

So, the field was used for `Method` and `Primitive`.